payload/recipes/ibexa_install.bash

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+cd $PROJECTMAPPINGFOLDER
+
+PHP="php"
```

Comment:
maybe we could be able to mutualize the DB ping thing with `ez_install.bash` recipe?

Reply:
+1